release_notes.py

```diff
@@ -179,9 +183,6 @@ def release_notes_section(self, store, title_prefix, current_server_version):
                     elif ans:
                         new_count += 1
                         new_str += ans
-                        print("Success")
```

Comment:
why did you remove this?

Reply:
all the success messages removed because it made the log garbage out.